### ISSUE TITLE
raintank-kubernetes-app plugin in not visible in grafana UI at plugin section

### ISSUE THREAD

**Issue by kmaiti** [raintank/kubernetes-app#54]
Hello,

As per details mentioned here [https://github.com/grafana/kubernetes-app#technical-details], we installed raintank-kubernetes-app but while we look at the UI[after restarting grafana], we don't see it in plugin app section. Ran grafana in debug mode but nothing is printed related to registering plugin. Any help is much appreciated.

Environment : Ran   Grafana v4.6.3 (commit: 7a06a47) in k8s environment [1.8]

Regards,
kamal

**Comment by daniellee** [comment]
How did you install it? If it is installed correctly, it should be in the data/plugins subdirectory.

**Comment by kmaiti** [comment]
@daniellee , Thank you for your response. I installed using helm. Default storagelocation was in value "storageLocalPath: /var/lib/grafana/data" as per helm chart. but plugin location I used "storageLocalPath: /var/lib/grafana/plugin".   Restarting command "service grafana-server restart" didn't help. But when I set  "storageLocalPath: /var/lib/grafana" then plugin was persisting and I had to delete running grafana pod. Grafana was able to register plugin. I saw following messages in console [ran in debug mode]

kubectl -n=monitoring logs grafana-grafana-6d4776758f-5ffbg |grep -i register

t=2017-12-20T12:34:54+0000 lvl=info msg="Registering plugin" logger=plugins name=Kubernetes
t=2017-12-20T12:34:54+0000 lvl=info msg="Registering plugin" logger=plugins name="Kubernetes Node Info"
t=2017-12-20T12:34:54+0000 lvl=info msg="Registering plugin" logger=plugins name="Kubernetes Pod Nav"
t=2017-12-20T12:34:54+0000 lvl=info msg="Registering plugin" logger=plugins name=kubernetes

and able to view raintank-kubernetes-app in UI of grafana. thanks for your hints.
